contract/src/tools/debug.js

```diff
@@ -1,41 +0,0 @@
-// @jessie-check
-
-let debugInstance = 1;
-
-/**
- * @param {string} name
- * @param {boolean | 'verbose'} enable
- */
-export const makeTracer = (name, enable = true) => {
```

Comment:
Are we getting `makeTracer` from somewhere else?

If we're just not using it currently, I suggest keeping it around for occasional usage.

Reply:
This a duplicate file - there is already another one in use.